### ISSUE TITLE
Translation ribbon button - "add responsibility" checkmark does not add the GitHub user id in the resp attribute

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-WriterBase#242]
see https://github.com/cwrc/CWRC-WriterBase/issues/123 for discussion.

**Comment by ilovan** [comment]
not sure if this has something to do wiith the GitHub API, @lucaju , but I suspect it does, since it passed muster once.

**Comment by ilovan** [comment]
same on prod, so no connection to the API changes

**Comment by ajmacdonald** [comment]
I looked into it and you need to be using a schema that supports the `resp` attribute on a `div` tag, e.g. the EpiDoc schema.

**Comment by ilovan** [comment]
you're right - can we then show the checkmark option only when the resp attribute is available?

**Comment by ajmacdonald** [comment]
Yes

**Comment by ajmacdonald** [comment]
df2bdbe509a002f34691c196d984bd5e9037bcf6

**Comment by ilovan** [comment]
confirmed on dev-cwrc-writer
